src/sentry/static/sentry/app/views/organizationEvents/events.jsx

```diff
@@ -102,7 +119,12 @@ class OrganizationEvents extends AsyncView {
 
         <EventsTable reloading={reloading} events={events} organization={organization} />
 
-        <Pagination pageLinks={eventsPageLinks} />
+        <div>
```

Comment:
Instead of floating `RowDisplay` left, let's make this div into a flex element and `justify-content: space-between` and `align-items: center`.